From: =?utf-8?q?David_Pr=C3=A9vot?= <taffit@debian.org>
Date: Wed, 19 Mar 2014 11:08:51 -0400
Subject: Use installed class for DEP-8 tests

Gbp: Topic DEP-8
---
 tests/bootstrap.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/bootstrap.php b/tests/bootstrap.php
index 8cb056a..62106a2 100644
--- a/tests/bootstrap.php
+++ b/tests/bootstrap.php
@@ -8,5 +8,4 @@ $loader = new ClassLoader();
 $loader->setUseIncludePath(true);
 $loader->register();
 $loader->addPrefixes(array(
-	'Sabre' => __DIR__.'/../lib',
 ));
